frontend/src/views/Pages/AccountOverview/GdtTransactionList.spec.js

```diff
@@ -0,0 +1,115 @@
+import { mount } from '@vue/test-utils'
+import GdtTransactionList from './GdtTransactionList'
+
+const localVue = global.localVue
+
+const apolloMock = jest.fn().mockResolvedValue({
+  data: {
+    listGDTEntries: {
+      count: 4,
+      gdtEntries: [
```

Comment:
I would set the gdtEntries in a own variable so we could Test if it is setten on the right variable or other Tests.
What do you think?

Reply:
We have three different types. They are all present and we have to test them all.